src/main/kotlin/racingcar/Round.kt

```diff
@@ -0,0 +1,12 @@
+package racingcar
+
+class Round {
+    var result = listOf<Car>()
+        private set
+
+    fun saveResult(carList: List<Car>) {
+        result = carList.map {
+            it.copy(it.distance)
+        }
+    }
```

Comment:
```suggestion
    private val _result = mutableListOf<Car>()
    val result: List<Car> get() = _result.toList()

    fun saveResult(carList: List<Car>) {
        _result.addAll(
            carList.map { it.copy(defaultDistance = it.distance) }
        )
    }
```
이와 같은 방식으로 컬렉션은 val로 사용할 수 있어요